tests/test_aggregate_verdict.py

```
@@ -1012,6 +1012,25 @@ def test_skip_counted_in_summary(self, tmp_path):
+                {
+                    "reviewer": "APOLLO",
+                    "perspective": "correctness",
+                    "verdict": "SKIP",
+                    "summary": "Review skipped due to timeout after 120s.",
+                }
+            )
+        )
+        (tmp_path / "athena.json").write_text(
+            json.dumps({"reviewer": "ATHENA", "perspective": "architecture", "verdict": "PASS", "summary": "Good."})
+        )
+        code, out, _ = run_aggregate(str(tmp_path))
```

Ruff (0.14.14)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)